net-misc/seafile-client/seafile-client-9.0.13.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Upstream is moving tags repeatedly, then we use commit hash.
+RELEASE_COMMIT="d11832792988ffeb2b514b693e3bcb39512da71e"
+
+inherit xdg cmake
+
+DESCRIPTION="Seafile desktop client"
+HOMEPAGE="https://www.seafile.com/ https://github.com/haiwen/seafile-client/"
+SRC_URI="https://github.com/haiwen/${PN}/archive/${RELEASE_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${RELEASE_COMMIT}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="shibboleth"
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-libs/jansson:=
+	dev-libs/openssl:=
+	dev-qt/qtbase:6[dbus,gui,network,widgets]
+	shibboleth? ( dev-qt/qtwebengine:6[widgets] )
+	dev-qt/qt5compat:6
+	>=net-libs/libsearpc-3.2.0_p1
+	~net-misc/seafile-${PV}
+	sys-libs/zlib
+	virtual/opengl
+	elibc_musl? ( sys-libs/fts-standalone )"
+DEPEND="${RDEPEND}"
```

Comment:
dev-libs/libevent is required too for seafile-client in DEPEND
https://github.com/haiwen/seafile-client/blob/d11832792988ffeb2b514b693e3bcb39512da71e/CMakeLists.txt#L390